.jenkins

```diff
@@ -96,6 +96,45 @@ cmake \
   -D Tasmanian_ENABLE_MPI=OFF \
   -D Tasmanian_ENABLE_FORTRAN=ON \
   -D Tasmanian_TESTS_OMP_NUM_THREADS=4 \
+.. &&
+                              make -j4 &&
+                              ctest -j4 -V --no-compress-output -T Test &&
+                              make install &&
+                              make test_install
+                        '''
+                    }
+                    post {
+                        always {
+                            xunit([CTest(deleteOutputFiles: true, failIfNotNew: true, pattern: 'build/Testing/**/Test.xml', skipNoTestFiles: false, stopProcessingIfError: true)])
+                        }
+                    }
+                }
+                stage('ROCm HIPCC') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.rocm'
+                            dir 'docker'
+                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-20.04:3.7'
+                            args '--device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video'
```

Comment:
Please set `HIP_VISIBLE_DEVICES` env variable
https://github.com/kokkos/kokkos/blob/03ded4e816da5927b946065b005ebfdf38e5a438/.jenkins#L34

Reply:
Actually can you see if `--env HIP_VISIBLE_DEVICES=${HIP_VISIBLE_DEVICES:-0}` works

Reply:
Added the variable.

The `HIP_VISIBLE_DEVICES` is not set by ArborX which I was using as a template:
https://github.com/arborx/ArborX/blob/31834fc818e32571b1f7666c9a8135b931b1d982/.jenkins#L336

Reply:
Yep need to fix. I am considering the introduction of `NVIDIA_VISIBLE_DEVICES` in our CUDA builds as well because of a new machine we are adding to the CI.

Reply:
I'll try `--env HIP_VISIBLE_DEVICES=${HIP_VISIBLE_DEVICES:-0}` too, waiting for `ctest` at the moment